folksonomy/models.py

```diff
@@ -90,3 +90,86 @@ class ValueCount(BaseModel):
 
 class PingResponse(BaseModel):
     ping: str
+
+
+class PropertyRenameRequest(BaseModel):
+    old_property: str
+    new_property: str
+
+    @field_validator("old_property", "new_property")
+    def property_check(cls, v):
+        if not v:
+            raise ValueError("property cannot be empty")
+        # strip the property
+        v = v.strip()
```

Comment:
do the strip before checking it's empty
```suggestion
        # strip the property
        v = v.strip()
        if not v:
            raise ValueError("property cannot be empty")
```